Server/ConcordiaCurriculumManager/Controllers/GroupController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using ConcordiaCurriculumManager.DTO;
+using ConcordiaCurriculumManager.Models.Users;
+using ConcordiaCurriculumManager.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ConcordiaCurriculumManager.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+
```

Comment:
For all the methods in the GroupController, can you add the Swagger-related attributes (see the other controllers for an example)